provisioning/device/samples/Getting Started/ComputeDerivedSymmetricKeySample/Parameters.cs

```diff
@@ -11,11 +11,11 @@ namespace Microsoft.Azure.Devices.Provisioning.Client.Samples
     internal class Parameters
     {
         [Option(
-            'd',
-            "DeviceId",
+            'r',
```

Comment:
@patilsnr I think this may be a reason the samples are passing sometimes in the pipeline. Between main and previews/v2 we had different parameters.

Reply:
I'll check this out. Thanks!